clang/lib/StaticAnalyzer/Checkers/SmartPtrModeling.cpp

```diff
@@ -853,7 +853,7 @@ void SmartPtrModeling::handleBoolConversion(const CallEvent &Call,
 
     const LocationContext *LC = C.getLocationContext();
     InnerPointerVal = C.getSValBuilder().conjureSymbolVal(
-        CallExpr, LC, InnerPointerType, C.blockCount());
+        Call, InnerPointerType, C.blockCount());
```

Comment:
For example, this inner pointer type is sometimes different from the call result type.